docker/thirdparties/docker-compose/elasticsearch/scripts/data/data1_es6.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
why add this file? es6 not support date format, it will be treat as long or string